src/SourceBuild/content/repo-projects/nuget-client.proj

```diff
@@ -15,6 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <EnvironmentVariables Include="NUGET_PACKAGES=$(NuGetPackageRoot)" />
```

Comment:
What does this have to do with test infrastructure?

Reply:
Before, we globally set NUGET_PACKAGES but we don't do that anymore so that we can have a separate nuget package cache for the VMR tests. Nuget-client is the only repo that doesn't respect the NUGET_PACKAGES env var that is provided by Arcade because they don't use Arcade: https://github.com/dotnet/arcade/blob/f54c598d137ab88bd473ea670b3c751be0540244/src/Microsoft.DotNet.Arcade.Sdk/tools/SourceBuild/SourceBuildArcadeBuild.targets#L90

I filed https://github.com/dotnet/source-build/issues/4293 which is relevant here. nuget-client incorrectly uses the VMR package cache (already in main).